rules/modular-monolith-testing/rule.md

```diff
@@ -1,7 +1,8 @@
 ---
-seoDescription: Learn the best testing strategies for modular monoliths, focusing on effective unit, integration, and workflow testing.
 type: rule
-title: Do you know how effectively test Modular Monoliths?
+title: Do you know how to effectively test Modular Monoliths?
```

Comment:
so if you look here, the sentence should say "Do you know how to effectively test Modular Monoliths?"